PF1/assignment6/MergeArrays.java

```diff
@@ -0,0 +1,48 @@
+package assignment6;
+
+/*This is the program to join two sorted array into a third sorted array.In this program 
+ * I assume that either the two sorted arrays are in ascending order or are n descending order and 
+ * returned array is in same order as in input array.
+ */
+public class MergeArrays {
+	int[] join(int a[], int a_size, int b[], int b_size, int c[]){
+		int i=0,j=0,k=0;
+		if(a_size==0)
+			return b;
+		else if(b_size==0)
+			return a;
+		
+		if(a[0]<=a[1]){
+			while(i<a_size & j<b_size){
+				if(a[i] <= b[j]){
+					c[k++] = a[i++];
+				}else{
+					c[k++] = b[j++];
+				}
+			}
+		
```

Comment:
Need to move code from line  24-30 out of if else so no need to write same code from line 39-44.